docs/source/release_notes/2022.1.rst

```diff
@@ -1,27 +1,114 @@
+FFT-accelerated matrix-vector multiplication routines that work on the Markov parameters directly.
+
+Structure-preserving model reduction for symplectic systems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+With `[#1621] <https://github.com/pymor/pymor/pull/1621>`_
+pyMOR now allows to formulate a quadratic Hamiltonian system as full-order model.
+Moreover, pyMOR implements multiple structure-preserving basis generation techniques
+`[#1600] <https://github.com/pymor/pymor/pull/1600>`_.
+In combination with a special reductor for quadratic Hamiltonian systems, a structure-preserving
+reduction, known as symplectic MOR, is available
+`[#1678] <https://github.com/pymor/pymor/pull/1678>`_. A demo script 
+for the linear wave equation is included.
 
 
 Additional new features
 ^^^^^^^^^^^^^^^^^^^^^^^
 
+Lots of new features for the neural network based reductors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In `[#1559] <https://github.com/pymor/pymor/pull/1559>`_, a couple of new features
+including support for learning rate schedulers, input and output scaling,
+regularization of weights and biases, a weighted MSE loss and logging of current
+losses have been added to improve the training of neural networks. These additions
+make the training process more flexible and all components can be combined as required.
+If no full-order model is available but only snapshot data, it is now also possible to
+use the neural network based reductors in a data-driven way.
+In `[#1679] <https://github.com/pymor/pymor/pull/1679>`_, the reductors have been
+adjusted to allow for pairs of parameters and corresponding snapshot data as training
+set instead of only parameters. The resulting surrogate models can be used as before.
+
+Randomized linear algebra algorithms
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+pyMOR's numerical linear algebra algorithms have been extended by randomized methods:
+the new :mod:`~pymor.algorithms.rand_la` module includes a method for
+:func:`singular value decomposition <pymor.algorithms.rand_la.random_generalized_svd>`
+of arbitrary linear pyMOR |Operators|, as well as an algorithm for solving
+:func:`generalized hermitian eigenproblems <pymor.algorithms.rand_la.random_ghep>`.
+The old randomized range approximation algorithms from the
+:mod:`~pymor.algorithms.randrangefinder` module have been moved to the new module as well
+`[#1552] <https://github.com/pymor/pymor/pull/1552>`_.
+
+FEniCS discretizer
+~~~~~~~~~~~~~~~~~~
+pyMOR's symbolic :class:`Expressions <pymor.analyticalproblems.expressions.Expression>`
+can now be converted to equivalent `UFL <https://fenics.readthedocs.io/projects/ufl/en/latest/>`_ 
+expressions. In particular, |ExpressionFunction| now has a `to_fenics` methods which
+utilizes this functionality under the hood `[#1550] <https://github.com/pymor/pymor/pull/1550>`_.
+Based on this feature an experimental :mod:`discretizer <pymor.discretizers.fenics>`
```

Comment:
Reference does not work, probably needs a rebase

Reply:
Right. I'll rebase and force push.